### ISSUE TITLE
Synchronizing Gives No Feedback for an Hour

### ISSUE THREAD

**Issue by zmintlimited** [NewYorkCoin-NYC/nycoin#26]
I am on a fast 1 gb fiber connection in New Zealand. The last NYC wallet took several days to synch, so I understand that it takes time. However for a new user it might appear that no progress is being made. An hour after installing the wallet, the first progress bar indicator just appeared.

The two-arrows icon in the lower right corner does not seem to be doing anything.

Can we show a count of how many blocks it is downloading and what block it is up to? Like Mist on Ethereum does? 

![screen shot 2018-07-26 at 6 31 45 pm](https://user-images.githubusercontent.com/29587555/43245494-8f90ff9e-9102-11e8-9d45-be3cdde9906f.png)

**Comment by MrSlosh** [comment]
I noticed the arrow spinner was not working either. I will have a look and see what's up.

I will poll other users to see if anyone else has noticed the sync progress bar not starting quickly. I haven't heard any reports just yet but may need to be a separate issue. Let's wait and see.

I believe while hovering over the progress bar or the spinning (supposed to be) arrows it pops up a tooltip and shows the progress.

**Comment by loopster40** [comment]
Can someone please help me. I am having trouble with my wallet constantly crashing after a long wait on syncing. I recently redid the wallet with the sync instructions by leaving the wallet.dat and the newyorkc.conf and adding the bootstrap file. It syncs for awhile and then crashes. Is there a node change that can be made or a connection change that can be made. Maybe a proxy setting that can be made?

**Comment by cmelion** [comment]
@loopster40 check out the support links at https://nycoin.community/  discord #wallets channel is your best bet.

**Comment by loopster40** [comment]
> @loopster40 check out the support links at https://nycoin.community/ discord #wallets channel is your best bet.

Where is the discord? That loads to the main page.

**Comment by cmelion** [comment]
scroll to the bottom to find social network links or use the menu in the top right to open the support link.

**Comment by loopster40** [comment]
So I got to go to a social network and search for help? Why does the main website not support issues with the wallet? The wallet has been junk ever since I downloaded it. I have been going through this for months and the wallet constantly crashed on me. I am doing everything they said to do even using the bootstrap. Sometimes runs for a few minuted then crashes.
________________________________

**Comment by cmelion** [comment]
This is not a support forum.
There are many people that are willing to help with your specific problem if you ask nicely.
